### DISCUSSION TITLE
What is the largest flying 🦜?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#927] [Galaxy Trivia] [answered]
**Bonus:** How long do they live?

**Comment by peterbaccaro** [comment, marked as answer]
The largest flying parrot is the **Hyacinth Macaw (Anodorhynchus hyacinthinus)**. With a length of about 100 centimetres (3.3 feet) from the head to the tip of the tail and a wingspan of approximately 150 centimetres (4.9 feet), it is considered the world's largest flying parrot species. The Hyacinth Macaw is known for its vibrant blue feathers, strong beak, and impressive size. These beautiful birds are native to the forests and grasslands of central and eastern South America, primarily found in Brazil, Bolivia, and Paraguay. Due to habitat loss and illegal pet trade, the Hyacinth Macaw is currently listed as vulnerable and conservation efforts are in place to protect their populations.

Hyacinth Macaws have a relatively long lifespan compared to many other parrot species. **In the wild, they can live up to 30 to 40 years**. However, in captivity, where they receive proper care, nutrition, and protection from predators, Hyacinth Macaws have been known to live even longer. **It is not uncommon for these parrots to reach 50 years of age or more in captivity.**
